examples/lighting-app/lighting-common/gen/endpoint_config.h

```diff
@@ -101,7 +101,7 @@
 #define ATTRIBUTE_SINGLETONS_SIZE (0)
 
 // Total size of attribute storage
-#define ATTRIBUTE_MAX_SIZE 3
+#define ATTRIBUTE_MAX_SIZE 6
```

Comment:
does this have a corresponding change in zap since this looks like generated code (in gen folder)?

Alternatively could we have a comment on how to define a 'correct' value?

Reply:
> does this have a corresponding change in zap since this looks like generated code (in gen folder)?

Not yet. At the moment `endpoint_config.h` is still coming from `AppBuilder` and not from ZAP. 
The PR to use ZAP to generate this file is #4077 

> Alternatively could we have a comment on how to define a 'correct' value?

Obviously no one should ever calculate that manually and it will be done automatically once #4077 lands.